### PR TITLE
Apply fixes from StyleCI

### PR DESCRIPTION
**Pull request by padraic**
This pull request applies code style fixes from an analysis carried out by [StyleCI](https://styleci.io).

---

For more information, click [here](https://styleci.io/analyses/qypO7o).

### CODE REVIEW

File: src/Report/Text.php [hunk 1]
```diff
@@ -69,7 +69,7 @@ public function prepareReportForMutant(Result $result)
     }
     
     /**
-     * 
+     *
```
Comment (pamil) [inline src/Report/Text.php]: This line should probably be removed.

**Comment by GrahamCampbell** [comment]
Not merged?

File: src/Report/Text.php (hunk 1 above)
Comment (GrahamCampbell) [inline src/Report/Text.php]: StyleCI can do this automatically if you enable more fixers than just those in the psr2 preset.

**Comment by padraic** [comment]
Reopen - At present these are manually created. Does it close these after subsequent analyses or something similar? I'll merge now.

**Pull request merged**